### PR TITLE
resource_labels added to the node_config nodepool

### PR DESCRIPTION
**Pull request by fulyagonultas**
Resource labels is needed in node_config block of nodepool module

---
**Checklist**

I applicable, I acknowledge that I have:
- [x] Read the [contributing guide](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md)
- [x] Ran `terraform fmt` on all modified files
- [x] Regenerated the relevant README.md files using [`tools/tfdoc.py`](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md#fabric-tools)
- [x] Made sure all relevant tests pass

### CODE REVIEW

**Comment by juliocc** [comment]
hi @fulyagonultas, we just had an internal discussion about this and we think the best approach is the following:
- Keep `var.labels` for what GKE resources call `resource_labels`. This is what we do in all other modules (including `gke-cluster-standard`) and we don't want to introduce different semantics here.
- Introduce a new variable for gke/k8s labels (e.g. `k8s_labels`) and use that for what GKE nodes call `labels`.

So in summary, keep var.labels as it is and add a new variable/field for kubernetes labels.

Can you make this changes? Sorry for mixup, but I didn't fully understand what you were trying to do initially.

**Review by ludoo — APPROVED** [review]

**Review by juliocc — APPROVED** [review]
